src/hooks/useTimer.ts

```diff
@@ -1,10 +1,8 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 
-// TODO: Abstract this as a props
-const mintDate = new Date('2022-05-23T00:00:00');
-
-export default function useTimer() {
+export default function useTimer(date: string) {
   const currentTime = Date.now();
+  const mintDate = useMemo(() => new Date(date), [date]);
```

Comment:
since this hook is a generic timer hook, let's rename `mintDate` to something more generic too - maybe `endDate`?

Reply:
Agree. Changed to `endDated` 🙌🏻